assets/js/alerts.js

```diff
@@ -184,8 +184,9 @@ function populateAlertTable(alerts) {
       const generatorLink = alert.generatorURL
         ? `<a href="${alert.generatorURL}" class="statusTileHealth" target="_blank">View</a>`
         : 'N/A'
+      const silencedAlert = alert.status.silencedBy.length > 0 ? 'silenced-alert' : 'active-alert'
```

Comment:
Could make a slight change here, set `silencedAlert` to either `silenced` or `active`, then can hard code the `-alert` into line 189 🙂

Reply:
Maybe also call it `alertStatus` to describe what it actually is.